services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_resource_tracker.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Annotated, Literal
+
+from fastapi import APIRouter, Depends, status
+from fastapi_pagination.api import create_page
+from models_library.api_schemas_resource_usage_tracker.credit_transactions import (
+    WalletTotalCredits,
+)
+from models_library.api_schemas_webserver.resource_usage import (
+    PricingPlanGet,
+    ServiceRunGet,
+)
+from models_library.resource_tracker import CreditTransactionId
+
+from ..models.pagination import LimitOffsetPage, LimitOffsetParamsWithDefault
+from ..models.resource_tracker_service_run import ServiceRunPage
+from ..services import (
+    resource_tracker_credit_transactions,
+    resource_tracker_pricing_plans,
+    resource_tracker_service_runs,
+)
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter()
+
+
+########
+# USAGE
+########
+
+
+@router.get(
+    "/usage/services",
+    response_model=LimitOffsetPage[ServiceRunGet],
+    operation_id="list_usage_services",
+    description="Returns a list of tracked containers for a given user and product",
+)
+async def list_usage_services(
+    page_params: Annotated[LimitOffsetParamsWithDefault, Depends()],
+    usage_services_page: Annotated[
+        ServiceRunPage,
+        Depends(resource_tracker_service_runs.list_service_runs),
+    ],
+):
+    page = create_page(
+        usage_services_page.items,
+        total=usage_services_page.total,
+        params=page_params,
+    )
+    return page
+
+
+######################
+# CREDIT TRANSACTIONS
+######################
+
+
+@router.post(
+    "/credit-transactions/credits:sum",
```

Comment:
```/transactions/credits:sum``` ?

Reply:
As we will have also "normal transactions" with the Payment service, I think it is not a bad idea to differentiate them as these are connected just to the osparc credits.